lib/cndb/omf.h

```diff
@@ -170,7 +170,7 @@ struct cndb_kvset_add_omf {
     uint64_t            kvset_add_dgen;
     uint64_t            kvset_add_vused;
     uint32_t            kvset_add_compc;
-    uint32_t            kvset_add_scatter;
+    uint32_t            kvset_unused; /* was scatter */
```

Comment:
@gsramdasi Left this CNDB OMF field in place, but renamed as unused.  Maybe we should remove it.

Reply:
Good to leave it as unused/rsvd field which preserves 8-byte alignment for this struct.